frontend/src/metabase/embedding/components/SharingMenu/MenuItems/ExportPdfMenuItem.tsx

```diff
@@ -12,6 +13,7 @@ import type { Dashboard } from "metabase-types/api";
 const handleClick = async (dashboard: Dashboard, includeBranding: boolean) => {
   const cardNodeSelector = `#${DASHBOARD_PDF_EXPORT_ROOT_ID}`;
   await saveDashboardPdf({
+    isEmbeddingSdk,
```

Comment:
since this is an env, you can import it right inside `saveDashboardPdf`, so you don't have to have an extra parameter.